tests/telemetry/test_telemetry.py

```diff
@@ -138,6 +162,19 @@ def test_telemetry_ouput(duthosts, rand_one_dut_hostname, ptfhost, setup_streami
     inerrors_match = re.search("SAI_PORT_STAT_IF_IN_ERRORS", result)
     pytest_assert(inerrors_match is not None, "SAI_PORT_STAT_IF_IN_ERRORS not found in gnmi_output")
 
+def test_osbuild_version(duthosts, rand_one_dut_hostname, ptfhost, localhost):
+    """ Test osbuild/version query.
+    """
+    duthost = duthosts[rand_one_dut_hostname]
+    cmd = generate_client_cli(duthost=duthost, method=METHOD_GET, target="OTHERS", xpath="osversion/build")
+    logger.debug("Command to run: {0}".format(cmd))
+    show_gnmi_out = ptfhost.shell(cmd)['stdout']
+    logger.debug(show_gnmi_out)
```

Comment:
This log is unnecessary. If log level set to DEBUG, the previous `ptfhost.shell(cmd)` call would log both invocation and result.